createdisk.sh

```diff
@@ -135,6 +135,7 @@ function update_json_description {
     diskSha256Sum=$(sha256sum $destDir/${CRC_VM_NAME}.qcow2 | awk '{print $1}')
 
     cat $srcDir/crc-bundle-info.json \
+        | ${JQ} ".name = \"${destDir}\"" \
```

Comment:
@guillaumerose I didn't understand the improvement using this patch, here we are putting the name against the `$destDir` which in the end use to create tarball for different platform, point I didn't get about ambiguity around doing same as part of  `crc` codebase.

Reply:
I would like when crc uses a bundle that it unpacks only the metadata to get the name, then create the directory, then unpack everything in without looking at the structure of the initial archive.

It will avoid bug when the crcbundle filename is different of the directory inside the crcbundle.

`crc start -p ~/pull-secret.txt -b ~/Downloads/crc_libvirt_2020-11-15\(5\).crcbundle --log-level debug`

I hope also to support both a path and a bundle name. `crc start -p ~/pull-secret.txt -b crc_libvirt_2020-11-15 --log-level debug`

Also, I would like to change the bundle name to `openshift-4.6.1`. So that I can have a collection of bundle like `openshift-x` and `okd-x`.

Reply:
> Also, I would like to change the bundle name to openshift-4.6.1. So that I can have a collection of bundle like openshift-x and okd-x.

This would make sense, but this would be `ocp` and `okd`, right? OpenShift is a very generic term in that case.